Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -2421,144 +2424,152 @@ protected ServerResponse doInBackground(Void... voids) {
         @Override
         protected void onPostExecute(ServerResponse serverResponse) {
             super.onPostExecute(serverResponse);
-            if (serverResponse != null) {
```

Comment:
The only change in this method is changing the `if` statement to reduce indentation layers, the change was from `if (serverResponse != null) {} ` to `if (serverResponse == null || isCancelled()) return`.